src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -2774,6 +2774,15 @@ enum BikesAllowed {
   NO_INFORMATION
 }
 
+enum CarsAllowed {
```

Comment:
I think this can go as well since it's not used anywhere.

Reply:
This is now removed